typo in comment

maybegeek opened the pull request:
typo in `var map` fixed. Just in a comment but for better readability `Post view` and `Main view` can be better distinguished.

maybegeek committed:
typo in comment

typo in `var map` fixed. Just in a comment but for better readability `Post view` and `Main view` can be better distinguished.

chrisdiana commented:
Thanks for catching this. I just merged a large commit that fixed some code standard stuff. Could you pull master, merge then resubmit the PR? Thanks!

maybegeek committed:
merge pull with code quality to pull request minor typo again

merge pull from upstream with code quality changes made,  to pull request minor typo again.

maybegeek commented:
Hi @cdmedia 

new pull request, see here:

(pull 26)

https://github.com/cdmedia/cms.js/pull/26

https://github.com/maybegeek/cms.js/commit/828d6476d95c0a8d4eddcfb6fdef370315747069
